mix.exs

```diff
@@ -4,7 +4,7 @@ defmodule Koans.Mixfile do
   def project do
     [app: :elixir_koans,
      version: "0.0.1",
-     elixir: ">= 1.2.1 and <= 1.4.0",
+     elixir: ">= 1.2.1 and <= 1.4.1",
```

Comment:
I have a hard time keeping the semantics of `~>` in my head, but I _think_ what you said is incorrect. I believe `~>1.4` is basically `1.x.y` where `x` >= `4`.

```
iex(1)> Version.match?("1.5.0", "~>1.4")
true
```

am i seeing that right?